circle.yml

```diff
@@ -6,7 +6,9 @@ general:
       - production
 machine:
   node:
-    version: 5.1.0
+    version: 5.11.0
+  svn:
```

Comment:
Subversion. That's a hack I used in `test_e2e.sh` to check out the client repositories, to avoid fetching the repo history which is big (only need HEAD).

http://stackoverflow.com/questions/9609835/git-export-from-github-remote-repository

Might otherwise be possible to achieve same effect with `git clone --depth=1`.